mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -177,11 +178,20 @@ func stop() error {
 			if err = deleteNs(utils.MesheryNamespace, client.KubeClient); err != nil {
 				return err
 			}
+			// Wait for the namespace to be deleted
+			done, err := mesheryNsDeleteCheck()
+			if err != nil {
+				log.Info(err)
+			} else if !done {
+				log.Info("Meshery is taking too long to stop.\nPlease check the status of the pods by executing “mesheryctl system status”.")
+			} else {
+				log.Info("Meshery is stopped.")
+			}
+		} else {
+			log.Info("Meshery is stopped.")
 		}
 	}
 
-	log.Info("Meshery is stopped.")
-
 	// Reset Meshery config file to default settings
 	if utils.ResetFlag {
```

Comment:
Let's shift this part of the code above because I feel this should be executed before the wait, the wait might return an error in some condition and then this won't be executed.

Let me know your thoughts!!

Reply:
We are not returning the error from `wait` any further, so it will still get executed (correct me if I'm wrong). If we move it above, we will have to move it before the deleting happens because this bit is common for docker and kubernetes setup.

Reply:
ahh, it will be okay now that we'll be logging in both cases above.

Reply:
So you reckon it's okay to move the config reset code above the delete code for both (k8s and docker)?

Reply:
let's leave it where it is. it would always execute because of logging. I'm already ending up introducing more and more bugs these days :laughing: